### PR TITLE
Update authentication.rst

### PR DESCRIPTION
**Pull request by taavit**
Added missing `use` base class for FoobarEncoder

### CODE REVIEW

**Comment by dupuchba** [comment]
:+1:

File: components/security/authentication.rst [hunk 1]
```diff
@@ -214,6 +214,7 @@ own, it just needs to follow these rules:
    method for this check::
 
        use Symfony\Component\Security\Core\Exception\BadCredentialsException;
+       use Symfony\Component\Security\Core\Encoder\BasePasswordEncoder;
```
Comment (xabbuh) [inline components/security/authentication.rst]: Can you please swap the two `use` statements to sort them alphabetically?

**Comment by weaverryan** [comment]
LOVE it. Thanks Dawid!

**Comment by weaverryan** [comment]
Btw, I accidentally merged this into 2.6, but just cherry-picked both commits to 2.3 at sha: 6f7e448f0b97a92504db7a7098708018d66fb277 and sha: cbb7ea136ab32638acaeecd8c39c26a713f7d2d3

Cheers!

**Pull request merged**